### ISSUE TITLE
Dropbox links apparently have bandwidth limits

### ISSUE THREAD

**Issue by kastnerkyle** [sklearn-theano/sklearn-theano#73]
@eickenberg What do you think we should do next? Know any hosting providers where paying ~10$ a month actually gets you somewhere? A little annoyed with this to be honest since I am paying for it and seems unlikely we are generating that much traffic.

**Comment by kastnerkyle** [comment]
200 GB/day is apparently not enough? I guess it is kind of a good thing but still I am not pleased with this. Looking ahead it seems like a few options are preset:

a) host at home (ugh)
b) host on S3 and have to build infrastructure to avoid DDOS on my wallet (ugh)
c) find another hosting provider or something that can act as an FTP server for 10$ a month.

**Comment by nouiz** [comment]
Host at MILA? What is the size?

**Comment by kastnerkyle** [comment]
MILA is a decent option though I would prefer to use something we have
admin over - will look into file sizes. I think each model is 500MB to 2GB
so they are fairly large... but I am not seeing other good options right
now. Looking through this at the moment - need to find one which has no
bandwidth limit and allows public links
http://en.wikipedia.org/wiki/Comparison_of_file_hosting_services

**Comment by nouiz** [comment]
I can give you access if needed. But I need to know the total current size
and expected futur size. This could make a big diff.

**Comment by kastnerkyle** [comment]
Yeah it is not clear... if you finish VGGNet support that is 3 models, then
it would be nice to have a caption model. I think it will get large - I
should see how Caffe "model zoo" does it on the backend - is it just random
hosted links or is there central repo?

**Comment by nouiz** [comment]
I didn't understood everything, but if the total data is less then 100G, we
have the space on our web server.

If it is more, I need to check, I think we can raise it easily, but I'm not
sure by how much.

The server can probably output at 30M/s or even more.

Fred
